### PR TITLE
OPENTOK-42100: Fix bug where tokbox site didn't load

### PR DESCRIPTION
**Pull request by v-kpheng**
#### What is this PR doing?
Fixes bug where sample app didn't load tokbox.com and showed an error message intead

#### How should this be manually tested?
- Clone this branch and build the sample app
- Run it
- Notice that tokbox.com site loads again

#### What are the relevant tickets?

Resolves [OPENTOK-42100](https://tokbox.atlassian.net/browse/OPENTOK-42100)

### CODE REVIEW

File: Screen-Sharing/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ When the app starts up, the `onCreate(Bundle savedInstanceState)` method instant
 object:
 
 ```java
-        mWebViewContainer = (WebView) findViewById(R.id.webview);
+mWebViewContainer = (WebView) findViewById(R.id.webview);
```
Comment (v-kpheng) [inline Screen-Sharing/README.md:14]: Cleaned up some formatting.

File: Screen-Sharing/README.md [hunk 2]
```diff
@@ -62,45 +62,42 @@ capturer supplies a new frame to the video stream. It creates a canvas, draws th
...
-    };
+    }
+};
 ```
 
 ## Next steps
 
 For details on the full OpenTok Android API, see the [reference
-documentation](https://tokbox.com/opentok/libraries/client/android/reference/index.html).
+documentation](https://tokbox.com/developer/sdks/android/).
```
Comment (v-kpheng) [inline Screen-Sharing/README.md:103]: Fixed this URL too.

File: Screen-Sharing/app/src/main/java/com/tokbox/android/tutorials/screen_sharing/MainActivity.java [hunk 3]
```diff
@@ -145,7 +145,7 @@ public void onConnected(Session session) {
         WebSettings webSettings = mWebViewContainer.getSettings();
         webSettings.setJavaScriptEnabled(true);
         mWebViewContainer.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
-        mWebViewContainer.loadUrl("http://www.tokbox.com");
+        mWebViewContainer.loadUrl("https://www.tokbox.com");
```
Comment (v-kpheng) [inline Screen-Sharing/app/src/main/java/com/tokbox/android/tutorials/screen_sharing/MainActivity.java:149]: Only secured URL gets redirected correctly.

**Comment by v-kpheng** [comment]
@IGitGotIt, please review.  Thanks!

**Review by igorwojda — APPROVED** [review]

**Comment by igorwojda** [comment]
Failing Travis checks can be ignored because travis was removed from repo and Github Actions will be added by this PR https://github.com/opentok/opentok-android-sdk-samples/pull/330

**Pull request merged**